pallets/valve/src/lib.rs

```diff
@@ -86,26 +104,68 @@ pub mod pallet {
 
 			ValveClosed::<T>::put(true);
 			<Pallet<T>>::deposit_event(Event::ValveClosed);
-			Ok(().into())
+			Ok(())
+		}
+
+		/// Tap the valve.
+		///
+		/// If valve is closed you cannot tap a pallet.
+		/// You cannot tap this paller, as then you could never untap it.
```

Comment:
when you say "tap" a pallet, you mean basically calling the pallet, right?

Reply:
idk the right word. A tapped pallet will not receive calls or whatever else we decide to turn off